File: OvalensPlanner/packages/shared/src/utils/currency.ts
export function formatGBP(amount: number): string {
  return new Intl.NumberFormat("en-GB", {
    style: "currency",
    currency: "GBP",
  }).format(amount);
}

export function formatRate(rate: number): string {
  return `${(rate * 100).toFixed(1)}%`;
}

File: OvalensPlanner/apps/api/app/utils/currency.py
"""Currency formatting utilities."""


def format_gbp(amount: float) -> str:
    """Format amount as GBP currency string."""
    return f"£{amount:,.2f}"


def format_rate(rate: float) -> str:
    """Format a decimal rate as a percentage string."""
    return f"{rate * 100:.1f}%"
